### ISSUE TITLE
Serenity reporting showing incorrect results

### ISSUE THREAD

**Issue by Unbr8kbal** [serenity-bdd/serenity-core#285]
Hi,

I'm running tests that i expect an exception to be thrown in.

However, as per attached screen shot, you'll see that 35 with errors, and in the Test Result Summary they come out as a Fail.
<img width="1071" alt="screen shot 2016-02-04 at 2 13 19 pm" src="https://cloud.githubusercontent.com/assets/1782545/12804759/e1491c76-cb49-11e5-8587-2cfa177b26a4.png">

The code looks like this
```
public class FooTest {
  @Rule
  public final ExpectedException exception = ExpectedException.none();

  @Test
  public void doStuffThrowsIndexOutOfBoundsException() {
    Foo foo = new Foo();

    exception.expect(IndexOutOfBoundsException.class);
    foo.doStuff();
  }
}
```
Thank you

**Comment by YamStranger** [comment]
Hello @Unbr8kbal!

Thanks for this issue!

Current version of reports was described in [manual](http://thucydides.info/docs/serenity-staging/#_sub_tab_test_count), your reports are correct: "error" have same meaning as fail in Test Result Summary.

Please close this issue, thanks!

**Comment by Unbr8kbal** [comment]
@YamStranger, 

You missed the main point, that test isn't suppose to fail or even error out.

**Comment by wakaleo** [comment]
Excepted exceptions are fairly unusual for automated acceptance tests - at this stage Serenity only supports the excepted attribute in the Test annotation, e.g.

@Test(expected=IndexOutOfBoundsException.class)
  public void doStuffThrowsIndexOutOfBoundsException() {...}

Out of curiosity, are these acceptance tests or a unit/integraton tests?

**Comment by Unbr8kbal** [comment]
@wakaleo ,
Integration tests, very interesting.

Since i've been reading up on this for a day i sore that way i.e. @Test(expected=) and when i ran it still seem to error out.

**Comment by wakaleo** [comment]
Fix released in version 1.1.32-rc.2
